### ISSUE TITLE
Ej 7 TP11

### ISSUE THREAD

**Issue by Jalbertonisalini** [marcelogarberoglio/PI#252]
Hola profe, quería saber si está bien está función map que me pide el ejercicio. No encontré testeos pero como parece cortita calculo que me podes decir rapidamente si esta bien o mal y por qué:

![imagen](https://github.com/marcelogarberoglio/PI/assets/127254155/b4ccf869-a052-46b0-a24d-9321d528853c)

**Comment by marcelogarberoglio** [comment]
¿Compila eso?  ¿list->first->tail es de tipo listADT?

**Comment by Jalbertonisalini** [comment]
Justamente me di cuenta de eso y no me diste tiempo a eliminarlo jajajaja.

**Comment by Jalbertonisalini** [comment]
Marcelo, si lo quiero hacer recursivo, y hago una mapRec que recibe Tlist list y la funcion ahi si estaba bien, no?

Y en map llamo a mapRec pasandole list->first y la funcion.

**Comment by marcelogarberoglio** [comment]
Sí, como todas las recursivas de un TAD: hay que llamar a una función auxiliar con el first.
